lib/mutant/integration/minitest.rb

```diff
@@ -0,0 +1,155 @@
+require 'minitest'
+
+# monkey patch for minitest
+module Minitest
+  # Prevent autorun from running tests when the VM closes
+  #
+  # Mutant needs control about the exit status of the VM and
+  # the moment of test execution
+  #
+  # @api private
+  #
+  # @return [nil]
+  def self.autorun
+  end
+
+end # Minitest
+
+module Mutant
+  class Integration
+    # Minitest integration
+    class Minitest < self
+      TEST_METHOD_PREFIX = 'test_'.freeze
+      TEST_FILE_PATTERN  = './test/**/{test_*,*_test}.rb'.freeze
+
+      register 'minitest'
+
+      # Compose a runnable with test method
+      #
+      # This looks actually like a missing object on minitest implementation.
+      class TestCase
+        include Adamantium, Concord.new(:klass, :test_method)
+
+        # Identification string
+        #
+        # @return [String]
+        def identification
+          "minitest:#{klass}##{test_method}"
+        end
+        memoize :identification
+
+        # Run test case
+        #
+        # @param [Object] reporter
+        #
+        # @return [Boolean]
+        def call(reporter)
+          ::Minitest::Runnable.run_one_method(klass, test_method, reporter)
+          reporter.passed?
+        end
+
+        # Cover expression syntaxes
+        #
+        # @return [Array<String>]
+        def expression_syntax
+          klass.cover_expression
+        end
+
+      end # TestCase
+
+      private_constant(*constants(false))
+
+      # Setup integration
+      #
+      # @return [self]
+      def setup
+        Pathname.glob(TEST_FILE_PATTERN)
+          .map(&:to_s)
+          .each(&method(:require))
+
+        self
+      end
+
+      # Call test integration
+      #
+      # @param [Array<Tests>] tests
+      #
+      # @return [Result::Test]
+      #
+      # rubocop:disable MethodLength
+      def call(tests)
+        test_cases = tests.map(&all_tests_index.method(:fetch)).to_set
+
+        output   = StringIO.new
+        reporter = ::Minitest::SummaryReporter.new(output)
+
+        start = Time.now
```

Comment:
Instead of recording `Time.now` and doing `Time.now - start` later could you use [`Benchmark.realtime`](http://ruby-doc.org/stdlib-2.2.4/libdoc/benchmark/rdoc/Benchmark.html#method-c-realtime)?